NoiseStudies/Macros/pRPDSignalDefinition.txt

```diff
@@ -0,0 +1,19 @@
+Positive RPD Ratio Cuts & Signal Definition
```

Comment:
Do you need this to run code or is this just for documentation?

Reply:
Not required / needed for code to run